### ISSUE TITLE
Access & Refresh tokens

### ISSUE THREAD

**Issue by Tsuintora** [tymondesigns/jwt-auth#1975]
### Subject of the issue

Hello guys, sorry for bad english. Thanks for your time.

How can i make Access token and Refresh tokens? Refresh token will refresh Access token. 
Need make access_token and refresh_token.
Thank you, i hope you understand this..

### Your environment
| Q                 | A
| ----------------- | ---
| Bug?              | no
| New Feature?      | no
| Framework         | Laravel
| Framework version | 7
| Package version   | 6.x.y
| PHP version       | 7.2

**Comment by rashivkp** [comment]
you can use access_token to refresh token even after expiry. 

`The user can refresh their token within a 2 week window of
  1     | the original token being created until they must re-authenticate.
  2     | Defaults to 2 weeks.
`

**Comment by usupsuparma** [comment]
I same issues with you, I need an access token with use refresh token to generate new access token

**Comment by amargoCactus** [comment]
I'm having the same issue

**Comment by fahmiegerton** [comment]
Yeah me too, my front-end app require refresh token included in response after login but couldn't find a way.

**Comment by brunogusta** [comment]
I have the same issue.
